app/views/includes/taxs_main.scala.html

```diff
@@ -61,8 +60,12 @@
     <script type="text/javascript" src='@routes.Assets.at("javascripts/details.polyfill.js")'></script>
 }
 
+@headScript = {
+    <script src="https://cdn.optimizely.com/js/@{ApplicationConfig.optimizelyProjectId}.js"></script>
```

Comment:
You can use the Optimizely Snippet in place of the generic headScripts https://github.com/hmrc/play-ui/blob/master/src/main/twirl/uk/gov/hmrc/play/views/layouts/OptimizelySnippet.scala.html 
An example of someone using that is at https://github.com/hmrc/tsp-import-vat-returns-frontend/blob/21bdbf06b0a189cd92b987effb9ec59b0212a070/app/views/templates/govuk_wrapper.scala.html#L52